go/libraries/doltcore/sqle/dtables/constraint_violations.go

```diff
@@ -49,44 +48,18 @@ func NewConstraintViolationsTable(ctx *sql.Context, tblName string, root *doltdb
 	} else if !ok {
 		return nil, sql.ErrTableNotFound.New(tblName)
 	}
-	sch, err := tbl.GetSchema(ctx)
+	cvSch, err := tbl.GetConstraintViolationsSchema(ctx)
```

Comment:
I moved it as it made more sense to create the schema on the table since it's used in more places than just the system tables. The other system tables only need a schema for their own use.